shaders/las_points.glsl

```diff
@@ -29,7 +29,7 @@ in vec3 color;
 in float distance;
 in int returnNumber;
 in int numberOfReturns;
-in int pointSourceId;
+in mediump int pointSourceId;
```

Comment:
Is this treating pointSourceId as a float, rather than an int?
Is _that_ why % modulo isn't working as expected?

Reply:
I think I was wrong about % not working. It seems to be fine now, I reverted those changes.